docker/nginx/conf.d/include/track-registry

```diff
@@ -1,27 +1,27 @@
 # register the registry access in accounts service (cookies should contain jwt)
 log_by_lua_block {
     -- this block runs only when accounts are enabled
-    if os.getenv("ACCOUNTS_ENABLED") ~= "true" then return end
+    if os.getenv("PORTAL_MODULES"):match("a") then
+        local function track(premature, request_method, jwt)
+            if premature then return end
 
-    local function track(premature, request_method, jwt)
-        if premature then return end
+            local httpc = require("resty.http").new()
+            local method = request_method == "GET" and "read" or "write"
```

Comment:
This is sooo weird. It merits a comment, at the very least. I've re-read it 5-6 times and I'm still not 100% how it works.

I guess it returns `read` when the HTTP method is `GET` but I don't know why. Maybe it's just how lua works. A short comment would be nice.

If we're storing the `read`/`write` operation in the variable then it should be called something else - `operation`/`action`/etc. If we call it `method` then I'll immediately expect it to be `GET`/`POST`/etc down in the code.

Reply:
so accounts api exposes tracking api for registry
- /track/registry/read
- /track/registry/write

Depending whether the actual client request was GET (registry read) or POST (registry write) we need to report to different api endpoint (above). So I'm creating `method` variable that contains either "read" or "write" string and concatenate it with the "/track/registry/<method_here>" to report to correct endpoint.

Reply:
And if you are asking about the syntax, it's a way to write a ternary in lua since lua has no native ternary operator

`local foo = true and "bar" or "baz"` - foo contains "bar"
`local foo = false and "bar" or "baz"` - foo contains "baz"

Reply:
I got the part with the different endpoint we want to hit. I guess my two points are:
* lua's ternary operator is a little unusual and we could use a comment explaining what's going on
* the name of the variable is misleading and we should rename it